src/styles.css

```
@@ -281,27 +301,43 @@ body { margin: 0; min-width: 320px; min-height: 100vh; overflow: hidden; }
 .pending-send { display: grid; grid-template-columns: minmax(0, 1fr) auto auto; align-items: center; gap: 6px; padding: 6px 8px; color: #4f3200; background: #fff4b8; border-top: 2px groove white; font: 12px/1.3 Tahoma, Arial, sans-serif; }
 .pending-send span { overflow: hidden; text-overflow: ellipsis; white-space: nowrap; }
 .pending-send .classic-button { min-height: 30px; padding: 3px 8px; }
-.transcript { min-width: 0; overflow-x: hidden; overflow-y: auto; padding: 14px 16px; background: #fff; }
+.transcript-shell { position: relative; min-width: 0; min-height: 0; overflow: hidden; }
+.transcript { width: 100%; height: 100%; min-width: 0; min-height: 0; overflow-x: hidden; overflow-y: auto; padding: 14px 16px; overscroll-behavior: contain; background: #fff; }
+.transcript-content { min-width: 0; contain: inline-size; }
+.new-messages-button { position: absolute; z-index: 3; right: 18px; bottom: 12px; max-width: calc(100% - 36px); min-height: 36px; padding: 6px 12px; overflow: hidden; color: #fff; background: var(--blue); border: 2px outset #fff; box-shadow: 2px 2px 0 rgba(0,0,0,.3); font: 700 13px/1.2 Tahoma, Arial, sans-serif; text-overflow: ellipsis; white-space: nowrap; }
+.new-messages-button:active { border-style: inset; }
 .message { display: grid; grid-template-columns: calc(92px * var(--transcript-magnification, 1)) 1fr; gap: 5px; margin-bottom: 10px; font-size: calc(17px * var(--transcript-magnification, 1)); line-height: 1.36; content-visibility: auto; contain-intrinsic-size: auto 54px; }
 .message time { white-space: nowrap; color: #222; font-family: "Courier New", Courier, monospace; font-size: calc(15px * var(--transcript-magnification, 1)); padding-top: 2px; }
 .message > div { min-width: 0; overflow-wrap: anywhere; word-break: break-word; }
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)